docs/sample_inventories/controller_broker_scram_kraft.yml

```diff
@@ -0,0 +1,49 @@
+---
+##
+## The following is an example inventory file of the configuration required for setting up Confluent Platform with Kraft Controller with SCRAM.
+## This will set up controller-controller communication via plain and other communication via scram. Please note: The value must be added at kafka_controller to achieve this behaviour.
+
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_user: ec2-user
+    ansible_become: true
+    ansible_ssh_private_key_file: /home/ec2-user/guest.pem
+    ansible_python_interpreter: /usr/bin/python3
+
+    ssl_enabled: true
+    sasl_protocol: scram
+
+kafka_controller:
+  vars:
+    kafka_controller_sasl_protocol: plain,scram
```

Comment:
Should we add something like following here and in next sample inventory as well ?
# How is it decided which sasl_protocol will be used for which communication.
# we should look at the variables from perspective of how their values would be defined in kafka_broker and kafka_controoler.
# in this case the variables arent defined in broker thus take the value from global. sasl_protocol: scram and kafka_controller_sasl_protocol: "{{sasl_protocol}}" thus kafka_controller_sasl_protocol: scram
# in controller it overrides the value of kafka_controller_sasl_protocol: plain,scram. 

# So broker-broker communication happens over scram as that is the value of sasl protocol in broker
# controller controller communication happens over plain as that is the first item in list of protocols on controller.
# controller-broker communication happens over scram as controllers as server have support for scram and brokers can become clients using their scram credentials.
cc: @mansisinha

Reply:
agreed with @mansi-jain-1206 , this is too much technical details for documentation

Reply:
cool.

Reply:
if not these technical details lets atleast add that defining kafka_controller_sasl_protocol inside kafka_controller and not all section is intentional here and it will be able to setup scram if we put that in all section. Typically controller variable defined in all or controller section both would create a similar cluster. so many would be under that impression and may unintentionally move it to all section if they dont see some warning here in comments atleast.

Reply:
moving it right above the line would better imv
also the value must be added at kafka_controller statement is a bit vague. maybe something like the variable kafka_controller_sasl_protocol must be defined in the kafka_controller section.